src/components/Footer/Footer.js

```diff
@@ -164,7 +163,7 @@ const Footer = ({
     try {
       refSheet.current.open();
       setLoading({...loading, loadingGetAllowAnonDmStatus: true});
-      const data = await getAllowAnonDmStatus(item.id);
+      const data = await getAllowAnonDmStatus('post', item.id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function call to `getAllowAnonDmStatus` now includes 'post' as a parameter. This change is likely made to accommodate changes in the function's signature or to provide more specific information to the function.
```diff
- const data = await getAllowAnonDmStatus(item.id);
+ const data = await getAllowAnonDmStatus('post', item.id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->